docs/new/posts/Percona Operators/operator-for-mysql-0.9.0.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2025-02-11
+description: >
+  Percona Operator for MySQL based on Percona Server for MySQL 0.9.0 has been released on Tuesday, February 11, 2025.
+authors: [fiowro]
+categories:
+  - Percona Operators
+tags:
+  - Percona Operator for MySQL
+---
+
+# Percona Operator for MySQL based on Percona Server for MySQL 0.9.0 has been released
+
+<!-- more -->
+
+[Percona Operator for MySQL based on Percona Server for MySQL](https://docs.percona.com/percona-operator-for-mysql/ps/index.html){:target="_blank"} 0.9.0 has been released on Tuesday, February 11, 2025.
+Version 0.9.0 of the Percona Operator for MySQL is still **a tech preview release**, and it is **not recommended for production environments**. You can try it out using the [Quickstart guide](https://docs.percona.com/percona-operator-for-mysql/ps/quickstart.html){:target="_blank"}.
+
+New features and improvements introduced by this release include:
+
+* [Scheduled backups](https://docs.percona.com/percona-operator-for-mysql/ps/backups-scheduled.md){:target="_blank"} are now supported in addition to on-demand ones
```

Comment:
https://docs.percona.com/percona-operator-for-mysql/ps/backups-scheduled.md -> This link results in 404 - Not found

Reply:
Thanks! Fixed!